.travis.yml

```diff
@@ -4,14 +4,20 @@ services:
   - docker
 
 before_script:
-  - docker-compose -v
+  - composer update
 
-install:
-  - docker-compose run --rm dev composer install
-  - docker-compose run --rm dev ./vendor/bin/phpcs --standard=psr2 --ignore=vendor -n .
-  - docker-compose run --rm dev ./vendor/bin/phpstan analyse ./src --level=max -c phpstan.neon
+script:
+  - composer ci
 
 php:
-  - 5.6
-  - 7.0
   - 7.1
+  - 7.2
+  - 7.3
+
+jobs:
+  include:
+    - php: 5.6
+      before_script:
+        - composer -dcomposer/php56/ install
```

Comment:
tohle možná šlo udělat jen pomocí --ignore-platform-reqs a nemusel jsi na to mít extra composer. Nebo to nějak dynamicky změnit ten composer.json - protože takhle nevím, jak to budeme udržovat. Budeš si vždycky pamatovat, že cokoli co je v require se musí přidat i do tohohle souboru?

Reply:
Možná jakože jednoduchý php skript 

```
$x = json_decode(file_get_contents())
unset($x['require_dev']);
$x['require_dev'] = [...];
$x['scripts'] = array_filter($x['scripts'], $fn);
file_put_contents(json_encode($x));
```
Prostě aby ten composer.json fakt odpovídal a bylo z něj odstraněné jen to co reálně není potřeba.

Reply:
Jelikož u php56 chceme pouštět jen ten phplint tak ten composer nebude potřeba nijak udržovat..

Reply:
No to právě bude. Co když přidáš do dependencies něco co nejde nahodit na 5.6?

Reply:
(ne do dev dependencies, ale něco co budeš používat v kódu)

Reply:
Ale jo, asi je to edge case, tak fajn. Ale tohle by mohl být vzorový repozitář, jak to dělat jinde.

Reply:
> (ne do dev dependencies, ale něco co budeš používat v kódu)

v tomhle máš pravdu

> Ale jo, asi je to edge case, tak fajn. Ale tohle by mohl být vzorový repozitář, jak to dělat jinde.

přidám to tam ještě

Reply:
dodělal jsem to tam

Reply:
Supr :)